src/darwin/Framework/CHIP/CHIPOperationalCredentialsDelegate.mm

```diff
@@ -237,6 +239,27 @@ static BOOL isRunningTests(void)
     return err;
 }
 
+static CHIP_ERROR AssignCATsForGivenDAC(const chip::ByteSpan & DAC, chip::CATValues & assignedCATs)
+{
+    uint16_t dacVid = VendorId::NotSpecified;
+    uint16_t dacPid = 0;
+
+    ReturnErrorOnFailure(ExtractDNAttributeFromX509Cert(MatterOid::kVendorId, DAC, dacVid));
+    ReturnErrorOnFailure(ExtractDNAttributeFromX509Cert(MatterOid::kProductId, DAC, dacPid));
+
+    if (IsTestVendorId(dacVid)) {
```

Comment:
I am entirely unconvinced we want this either.  Why do we want it here?

Reply:
The implementation was changed